### PR TITLE
Added Images and inputs

### PR DESCRIPTION
**Pull request by syed-shaida-hussain**
[Images live](https://dazzling-lalande-fabe9e.netlify.app/image/image)
[Inputs live](https://dazzling-lalande-fabe9e.netlify.app/input/input)

### CODE REVIEW

**Review by sanjay-xdr — APPROVED** [review]
everthing looks good

File: Image/image.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    .responsive-img{
+        max-width: 18rem;
+    }
+}
+
+@media only screen and (max-width:300px){
+    .responsive-img{
+        max-width: 10rem;
+    }
+}
```
Comment (xii-vi) [inline Image/image.css]: Try to minimize the media queries. Use a maximum of three.

File: Image/image.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            <h1 class="image-heading">Images</h1>
+            <p class="image-info">
+                Images are the informative and emotionally appealing layout elements that are seen and scanned first.
+            </p>
+
+            <div class="overlay-text">I am a responsive image.</div>
+
+            <div class="images">
+
+                <img class="responsive-img" src="/images/ecommerce-landing-pic.svg" alt="">
```
Comment (xii-vi) [inline Image/image.html:59]: provide alternate if possible

File: Image/image.css (hunk 1 above)
Comment (syed-shaida-hussain) [reply Image/image.css]: ok , will do it.

**Review by xii-vi — APPROVED** [review]
Resolve the issue mentioned in the comment, after that you can merge.

**Pull request merged**